scripts/globals/mobskills/PW_Gorgon_Dance.lua

```diff
@@ -1,31 +1,32 @@
 ---------------------------------------------
--- Gorgon Dance
+--  Gorgon Dance
 --
--- Description: Petrifies all targets in an area of effect.
--- Type: Enfeebling
--- Utsusemi/Blink absorb: Ignores shadows
--- Range: 10' radial?
--- Notes: Used only by Medusa. Starts using it at 25%.
+--  Description: Petrifies all targets in an area of effect.

```

Comment:
@nesstea  Why'd  you  bother  with  changing  the  spacing ? 
Single space was correct formatting anyway, not that anybody cares whats in the headers or we wouldn't have 40 diff versions among the skill scripts.